pagopa/io-app
# chore: [IOBP-496] Complete removal of SV

## Short description
deleted `bonus/siciliaVola` folder, along with some cleanup here and there

## List of changes proposed in this pull request
- deleted folder
- moved shared assets to a common folder
- removed package.json entries
- removed unused I18n keys --along with non-SV ones, after receiving confirmation from @CrisTofani 
- removed remaining references around the app 

## How to test
while I risk sounding like a broken record, 
- `rm -r ./definitions` 
this is kind of vital since the definitions  are deleted by the single scripts, which have been deleted themselves.
- `cp .env.local .env`
- `yarn generate`
- have a look around in the app

as always, nothing should break on the user's side and the expected outcome is to not notice any difference from the master branch.
Most of all, make sure the airBaloon and timeout pictograms render correctly, since it is the only "non removal" change
